src/c2d.jl

```diff
@@ -42,31 +41,31 @@ end
   Cd    = C
   Dd    = D + C*M2
   x0map = [speye(nx) -M2]
-  StateSpace{Siso{T}}(Ad, Bd, Cd, Dd, Ts), x0map
+  ss(Ad, Bd, Cd, Dd, Ts), x0map
```

Comment:
I will check this part. This was Cristian's implementation, enforcing some SISO/MIMO structure regardless of the type of `D` matrix. His version was correct. I'll fix this.